src/ProjectTemplates/Web.ProjectTemplates/content/Worker-FSharp/.template.config/template.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    "Worker",
+    "Web"
+  ],
+  "name": "Worker Service",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty project template for creating a worker service.",
+  "groupIdentity": "Microsoft.Worker.Empty",
+  "precedence": "7000",
```

Comment:
@ryanbrandenburg What happens when multiple templates have the same precedence?

Reply:
It doesn't matter that they have the  same precedence so long as their `groupIdentity` is different. Precedence is basically a tie-breaker for if you have multiple templates installed with the same `groupIdentity`, otherwise you have to have a parameter which differentiates them (such as TFM).

Reply:
Thanks!